tests/datamodules/test_utils.py

```diff
@@ -43,16 +46,21 @@ def test_group_shuffle_split() -> None:
         match=re.escape("`train_size` and `test_size` must be in the range (0,1)."),
     ):
         group_shuffle_split(groups, train_size=-0.2, test_size=1.2)
-    with pytest.raises(ValueError, match="26 groups were found, however the current *"):
+    with pytest.raises(ValueError, match="3 groups were found, however the current *"):
         group_shuffle_split(groups, train_size=None, test_size=0.999)
 
-    train_indices, test_indices = group_shuffle_split(
-        groups, train_size=None, test_size=0.2
-    )
-    assert len(set(train_indices) & set(test_indices)) == 0
-    assert len(set(groups[train_indices])) == 21
-    train_indices, test_indices = group_shuffle_split(
-        groups, train_size=0.8, test_size=None
-    )
-    assert len(set(train_indices) & set(test_indices)) == 0
-    assert len(set(groups[train_indices])) == 21
+    test_cases = [(None, 0.2, 42), (0.8, None, 42)]
+
+    for train_size, test_size, random_state in test_cases:
+        train_indices1, test_indices1 = group_shuffle_split(
+            groups,
+            train_size=train_size,
+            test_size=test_size,
+            random_state=random_state,
+        )
+        # Check that the results are the same as the saved results
```

Comment:
```suggestion
        # Check that the results are the same as expected
```
We are no longer "saving" them per se